src/lib/ymodem.js

```diff
@@ -101,14 +101,17 @@ module.exports = class YModem {
 				}
 
 				// wait for initial response
+				const readyMsg = "Waiting for the binary file to be sent ... (press 'a' to abort)";
+				const linebreakPtn = /\r?\n/;
 				let line = '';
 				function cmdResponse(){
 					let data = self.port.read();
 					self._logData(data);
 					line += data.toString();
+					const lines = line.split(linebreakPtn);
```

Comment:
Ah, that's a tricky one. Good thinking that data packets may include many lines.